django_recaptcha/static/django_recaptcha/js/widget_v2_invisible.js

```diff
@@ -0,0 +1,49 @@
+(function () {
+
+  function main() {
+    for (const captchaElement of document.querySelectorAll(".g-recaptcha")) {
+
+      const widgetUuid = captchaElement.getAttribute("data-widget-uuid");
+      if (!widgetUuid) {
+        console.warn("reCAPTCHA widget with missing UUID");
+        break;
+      }
+      console.log(`found reCAPTCHA widget with UUID '${widgetUuid}'`);
+
+      const callbackFunctionName = captchaElement.getAttribute("data-callback");
+      if (!callbackFunctionName) {
+        console.warn(`callback function missing from reCAPTCHA widget with UUID '${widgetUuid}'`);
+        break;
+      }
+      if (window.hasOwnProperty(callbackFunctionName)) {
+        console.warn(`callback function '${callbackFunctionName}' has already been added`);
+        break;
+      }
+
+      const formElement = captchaElement.closest("form");
+      if (!formElement) {
```

Comment:
I believe this could just be:

```suggestion
      if (! captchaElement.form) {
```